lib/converter.js

```diff
@@ -75,8 +91,19 @@ async function convert(inputFile, options, timings) {
     if (format) {
       pdfOptions.format = format // Paper format. If set, takes priority over width or height options. Defaults to 'Letter'.
     }
-    return await page.pdf(pdfOptions)
+    await page.pdf(pdfOptions)
   } finally {
+    if (outputToStdout && await existsFile(outputFile)) {
+      console.log(await readFile(outputFile, 'utf-8'))
+      await removeFile(outputFile)
+    }
+    // QUESTION: should we remove the temporary HTML file ?
+    // It useful when building a new layout but the converter should only output a PDF file.
```

Comment:
I use temp html file for live preview: I open it in browser with autoreload. I think I can use something similar with PDF.

Perhaps we need to delete html by default, but then, for html-preview specifically, have a flag `--generate-only-html`?

OTOH, if PDF generation is not significanly slower then html generation, perhaps I can adjust my workflow to preview PDFs direclty?

Reply:
> I use temp html file for live preview: I open it in browser with autoreload. I think I can use something similar with PDF.

Indeed but the major drawback is that it can be harder to debug/prototype.
I'm using the dev tools a lot when I create a new layout.

> Perhaps we need to delete html by default, but then, for html-preview specifically, have a flag --generate-only-html?

Why not...
The HTML page is an intermediate step so maybe `--keep-html` ?

> OTOH, if PDF generation is not significantly slower then html generation, perhaps I can adjust my workflow to preview PDFs directly?

It depends on the size or your document but the PDF generation is relatively fast.